packages/lib-user/src/components/UserStats/components/ProfileHeader.spec.js

```diff
@@ -0,0 +1,87 @@
+import { render, screen } from '@testing-library/react'
+import { composeStory } from '@storybook/react'
+
+import Meta, { Default, Group, GroupWithHours, UserWithHours } from './ProfileHeader.stories.js'
+
+describe('components > UserStats > ProfileHeader', function () {
+  describe('with a user', function () {
+    it('should show the user\'s avatar', function () {
+      const DefaultStory = composeStory(Default, Meta)
+      render(<DefaultStory />)
+      const avatar = screen.getByAltText('TestUser avatar')
+      expect(avatar).to.be.ok()
+    })
+
+    it('should show the user\'s display name', function () {
+      const DefaultStory = composeStory(Default, Meta)
+      render(<DefaultStory />)
+      const displayName = screen.getByText('Test User')
+      expect(displayName).to.be.ok()
+    })
+
+    it('should show the user\'s login', function () {
+      const DefaultStory = composeStory(Default, Meta)
+      render(<DefaultStory />)
+      const login = screen.getByText('@TestUser')
+      expect(login).to.be.ok()
+    })
+
+    it('should show the user\'s classifications', function () {
+      const DefaultStory = composeStory(Default, Meta)
+      render(<DefaultStory />)
+      const classifications = screen.getByText('384')
+      expect(classifications).to.be.ok()
+    })
+
+    it('should show the user\'s projects', function () {
+      const DefaultStory = composeStory(Default, Meta)
+      render(<DefaultStory />)
+      const projects = screen.getByText('22')
+      expect(projects).to.be.ok()
+    })
+
+    it('should show the user\'s hours', function () {
+      const UserWithHoursStory = composeStory(UserWithHours, Meta)
+      render(<UserWithHoursStory />)
+      const hours = screen.getByText('5')
+      expect(hours).to.be.ok()
+    })
+  })
+
+  describe('with a group', function () {
+    it('should show the group\'s display name', function () {
+      const GroupStory = composeStory(Group, Meta)
+      render(<GroupStory />)
```

Comment:
```suggestion
  describe('with a group', function () {
    const GroupStory = composeStory(Group, Meta)
    
    it('should show the group\'s display name', function () {
      render(<GroupStory />)
```

Same here, `GroupStory` only needs to be defined once in the `describe()` block, and the rest can be deleted in the `it()` statements.